source/addonHandler/__init__.py

```diff
@@ -427,31 +427,45 @@ def getAvailableAddons(
 def installAddonBundle(bundle: AddonBundle) -> Addon | None:
 	""" Extracts an Addon bundle in to a unique subdirectory of the user addons directory,
 	marking the addon as needing 'install completion' on NVDA restart.
+
+	:param bundle: The add-on bundle to install.
+	The bundle._installExceptions property is modified to store any raised exceptions
+	during the installation process.
+
+	:return: The installed add-on object, or None if the installation failed.
+	Regardless if installation failed or not, the created add-on object should be returned
```

Comment:
Seems this line contradicts the previous line? I.e. returning None if installation fails verses the addon object should ways be returned even if installation fails.